src/libcollections/btree/map.rs

```diff
@@ -276,6 +344,14 @@ pub struct OccupiedEntry<'a, K: 'a, V: 'a> {
 
 impl<K: Ord, V> BTreeMap<K, V> {
     /// Makes a new empty BTreeMap with a reasonable choice for B.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::collections::BTreeMap;
+    ///
+    /// let mut map: BTreeMap<&str, isize> = BTreeMap::new();
```

Comment:
>  removed it for consistency.

Since I haven't gone through any of the collections yet, we can invent our own consistency, and change them all. It's okay if they're a bit inconsistent in the meantime.